ndc-test/src/main.rs

```diff
@@ -71,15 +85,19 @@ async fn main() {
                 headers: HeaderMap::new(),
             };
 
-            let results =
-                ndc_test::test_snapshots_in_directory(&configuration, snapshots_dir).await;
+            let mut reporter = ConsoleReporter::new();
 
-            if !results.failures.is_empty() {
-                println!();
-                println!("{}", report(results));
+            ndc_test::test_snapshots_in_directory(&configuration, &mut reporter, snapshots_dir)
+                .await;
 
-                exit(1)
-            }
+            // let results = reporter.results();
+
+            // if !results.failures.is_empty() {
+            //     println!();
+            //     println!("{}", report(results));
+
+            //     exit(1)
+            // }
```

Comment:
Are these comments worth keeping around?

Reply:
Oh, this needs adding back.